include/bbp/sonata/report_reader.h

```diff
@@ -138,7 +138,7 @@ class SONATA_API ReportReader
         std::pair<size_t, size_t> getIndex(const nonstd::optional<double>& tstart,
                                            const nonstd::optional<double>& tstop) const;
 
-        std::map<NodeID, std::pair<uint64_t, uint64_t>> nodes_pointers_;
+        std::map<NodeID, Selection::Range> nodes_pointers_;
```

Comment:
I would use a local type, not the `Selection::Range`, as its behavior might change.  Semantically they operate the same now, but that's not guaranteed to be the case; this isn't really a `Selection`

Reply:
Agreed, it was just to avoid duplicating code, but we can change it.